src/main/java/com/google/sps/servlets/AddDataServlet.java

```diff
@@ -0,0 +1,45 @@
+package main.java.com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import main.java.com.google.sps.data.Donation;
+import main.java.com.google.sps.data.Selection;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/new-data")
+public class AddDataServlet extends HttpServlet {
```

Comment:
rename data service to what it actually does here which is add donation. In the same sense I think the servlet address should be /donation/new

Reply:
And the class can be AddDonationServlet

Reply:
this function is for testing collaboration, not adding donation, so I try to add some data in donation database. And also this PR is just a draft and I am not going to merge it. Team members asked me to push it to show the basic structure of folders and files.